interfaces/Portal/src/app/app-routing.module.ts

```diff
@@ -130,7 +122,7 @@ const routes: Routes = [
         // Fallback for change in url, from old to new syntax:
         path: 'registration-validation',
```

Comment:
```suggestion
        path: ProgramPhase.registrationValidation,
```

I think we can remove this old-old URL-path and use this redirect for the "previous-old" URL-path.

So users' still-open-browser-tabs and bookmarks will continue to work...